service/controller/v20/resource/node/create.go

```diff
@@ -80,7 +79,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	// associated control plane pod, we delete it from the tenant cluster's
 	// Kubernetes API.
 	for _, n := range nodes {
-		if node.IsNodeReady(&n) {
```

Comment:
This function is no longer exported upstream (see https://github.com/kubernetes/kubernetes/pull/73656/files), so it was forked/inlined.